antarest/core/utils/utils.py

```diff
@@ -61,6 +61,24 @@ def sanitize_uuid(uuid: str) -> str:
     return uuid
 
 
+def validate_study_name(name: str) -> str:
```

Comment:
We could add a small docstring to explain why:
- / to not conflict with folders, I guess
- = to be written safely in INI files, I think ...